spec/forms/risks/risk_from_others_spec.rb

```diff
@@ -0,0 +1,95 @@
+require 'rails_helper'
+
+RSpec.describe Forms::Risks::RiskFromOthers, type: :form do
+  let(:model) { Risks.new }
+  subject { described_class.new(model) }
+
+  let(:params) {
+    {
+      rule_45: 'yes',
+      rule_45_details: 'Rule 45 present',
+      csra: 'high',
+      csra_details: 'High risk of CSRA',
+      verbal_abuse: 'yes',
+      verbal_abuse_details: 'Many verbal abuses',
+      physical_abuse: 'yes',
+      physical_abuse_details: 'some physical abuses',
+    }.with_indifferent_access
```

Comment:
does the class you've designed use stringy keys or symbols? indifferent access hides the design you've chosen